server/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM php:7.2-apache
+
+RUN apt-get update \
+    && apt-get install -y \
+       libpng-dev \
+       libpq-dev \
+       libsqlite3-dev \
+       libxml2-dev \
+       libzip-dev \ 
+       git \
+       sqlite \
+       libfreetype6-dev \
+       libjpeg62-turbo-dev \
+    && apt-get autoremove -y \
+    && apt-get autoclean \
+    && apt-get clean \
+    && rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/* \
+    && docker-php-ext-install -j$(nproc) \
+       opcache \
+       posix \
+       simplexml \
+       zip \
+    && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
+    && docker-php-ext-install -j$(nproc) gd 
+
+WORKDIR /var/www/html
+
+RUN git clone https://github.com/nextcloud/server.git nc
```

Comment:
Why are you cloning to `nc` and not to the working dir directly. Then there wouldn't be a need to have it in a subdir like `localhost/nc` ;)